### PR TITLE
prevent wrapping in guilabel role

### PR DESCRIPTION
**Pull request by hannydevelop**
This is for issue #269. Since I was fixing some CSS already for #248, I just did this too. @ronso0 and @Holzhaus what do you think?, this is ready for review.

### CODE REVIEW

**Comment by hannydevelop** [comment]
no-wrap has been effected as shown in the picture below:
Mixxx2

File: source/_static/css/mixxx.css [hunk 1]
```diff
@@ -157,6 +157,8 @@ a:hover {
 }
 .rst-content .guilabel {
     background: #faf2e7;
+    white-space: nowrap !important;
```
Comment (Holzhaus) [inline source/_static/css/mixxx.css:160]: Is `!important` really necessary?

Comment (hannydevelop) [reply source/_static/css/mixxx.css:160]: not really, would remove that now, was checking why `white-space: nowrap` didn't work at first; didn't commit my work.
I'll remove this now

**Comment by hannydevelop** [comment]
Hello @Holzhaus done! please review!

**Review by Holzhaus — APPROVED** [review]
LGTM, thank you.

**Pull request merged**